packages/alignments/package-lock.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "@gmod/jbrowse-plugin-alignments",
```

Comment:
We don't want to keep package-lock.json files...may want to create a gitignore for it also if this was accidentally committed

Reply:
yeah I think that was commited on accident during one of my pushes

Reply:
no worries